### PR TITLE
forced reload on project nav bar click

### PR DESCRIPTION
**Pull request by blackmacaroon**
# Description

Describe change or new feature here.

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -1,15 +1,20 @@
 import React from "react"
+import {connect} from 'react-redux';
```
Comment (lappjeff) [inline src/components/NavBar.js]: Are we using Redux in this at all or can we remove the import and comments?

File: src/components/ProjectCard/ProjectSearch.js [hunk 2]
```diff
@@ -15,40 +15,45 @@ const ProjectSearch = props => {
...
     }
     //eslint-disable-next-line
-  }, [])
+  }, [projectLength])
 
   const handleSearch = (e, offset, searchString) => {
-    e.preventDefault()
+    if(e) e.preventDefault()
     searchProjects(offset, searchString)
+    // console.log(projectLength)
```
Comment (lappjeff) [inline src/components/ProjectCard/ProjectSearch.js]: Comments not welcome here 🔥

File: src/components/NavBar.js (hunk 1 above)
Comment (blackmacaroon) [reply src/components/NavBar.js]: that was left over from a failed attempt

Comment (blackmacaroon) [reply src/components/NavBar.js]: yeet

File: src/components/ProjectCard/ProjectSearch.js (hunk 2 above)
Comment (blackmacaroon) [reply src/components/ProjectCard/ProjectSearch.js]: shwoopsie. yeet.

**Pull request merged**